lib/rubycritic/generators/html_report.rb

```diff
@@ -18,6 +18,7 @@ def generate_report
         create_directories_and_files
         copy_assets_to_report_directory
         puts "New critique at #{report_location}"
+        report_location
```

Comment:
because it want to get report path before report will be open in Command::Default class , so there is another way?